src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java

```diff
@@ -459,6 +465,11 @@ public Node reconfigure(final StaplerRequest req, JSONObject form) throws FormEx
         return null;
     }
 
+    @Override
+    public boolean isAcceptingTasks() {
```

Comment:
From the javadocs this signals that the computer can't accept tasks so when terminating once out of the lock this node should not be able to receive a task